gst/nnstreamer/tensor_common.h

```diff
@@ -174,6 +174,17 @@ gst_tensor_time_sync_buffer_from_collectpad (GstCollectPads * collect, tensor_ti
 extern GstCaps *
 gst_tensor_pad_caps_from_config (GstPad * pad, const GstTensorsConfig * config);
 
+/**
+ * @brief Get all possible caps from tensor config. Unlike gst_tensor_pad_caps_from_config(), this function does not check peer caps.
```

Comment:
This is a trivial one.
```suggestion
 * @brief Get all possible caps from tensors config. Unlike gst_tensor_pad_caps_from_config(), this function does not check peer caps.
```

Reply:
updated